scripts/tools/KvToRdsMigration.php

```diff
@@ -262,11 +262,10 @@ protected function provideUsage()
      */
     protected function provideDescription()
     {
-        return __(
-            'A script to migrate result from KeyValue storage to RDS.' . PHP_EOL
-            . ' It copies data from KeyValue to RdsStorage and switch config from '
-            . 'taoResultServer/resultservice.conf.php.'
-        );
+        // phpcs:disable Generic.Files.LineLength
+        return __('A script to migrate result from KeyValue storage to RDS.' . PHP_EOL .
```

Comment:
It is still line breaking, the translation will not apply

Reply:
Ok, this was already wrong before our refactoring https://github.com/oat-sa/extension-tao-outcomerds/blob/master/scripts/tools/KvToRdsMigration.php#L265